lib/bitcoin/BitcoinRawDataParser.ts

```diff
@@ -0,0 +1,85 @@
+import { Block } from 'bitcore-lib';
+
+/**
+ * Parser for raw bitcoin block data
+ */
+export default class BitcoinRawDataParser {
+
+  /**
+   * The beginning of each block contains the magic bytes indicating main or test net
+   * followed by a 4 byte number indicating how big the block data is
+   */
+  private static magicBytes = {
+    testnet: '0b110907',
+    mainnet: 'f9beb4d9'
+  };
+  private static sizeBytesLength = 4;
+
+  /**
+   * Parse the given raw block data file. It can throw error if block data is invalid when validating magic bytes,
+   * creating new Block, or validating size
```

Comment:
Can you add here a quick digest of example what the block data look like? I am not sure I understand the format looking at the loop...

e.g. <magic bytes><block1 size><block1 data><block2 size><block2 data>...

Reply:
Ya I will add documentation. For your refernce now, this is the block data 
https://user-images.githubusercontent.com/26752423/76470914-8c729400-63ae-11ea-882f-c5919ee78305.png